data/restapi/src/main/java/tech/pegasys/artemis/restapi/RestApi.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.restapi;
```

Comment:
restapi is a little generic considering there could be a multiple restapi's (e.g. validator, datarestapi, controlrestapi).  names are hard, but you could consider calling it `tech.pegasys.artemis.data.restapi` or `tech.pegasys.artemis.datarestapi`

Reply:
agreed